AnalyticsKit.podspec

```diff
@@ -20,18 +20,38 @@ Pod::Spec.new do |s|
 
 
 
-  s.platform     = :ios, '8.0'
+  s.platform     = :ios, '8.4'
   s.source       = { :git => "https://github.com/twobitlabs/AnalyticsKit.git", :tag => s.version.to_s }
   s.requires_arc = false
 
   s.subspec 'Core' do |core|
-    core.source_files  = 'AnalyticsKit.swift', 'AnalyticsKitEvent.{h,m}', 'AnalyticsKitDebugProvider.swift', 'AnalyticsKitUnitTestProvider.{h,m}', 'Categories/NSNumber+Buckets.{h,m}', 'AnalyticsKit/AnalyticsKit/AnalyticsKitTimedEventHelper.{h,m}'
+    core.source_files  = 'AnalyticsKit.swift', 'AnalyticsKitEvent.swift', 'AnalyticsKitDebugProvider.swift', 'AnalyticsKitUnitTestProvider.swift', 'AnalyticsKit/AnalyticsKit/AnalyticsKitTimedEventHelper.swift'
   end
 
-  s.subspec 'Crashlytics' do |a|
-    a.source_files = 'Providers/Crashlytics/AnalyticsKitCrashlyticsProvider.swift'
-    a.dependency 'Crashlytics'
-    a.dependency 'AnalyticsKit/Core'
+  s.subspec 'Intercom' do |i|
+    i.source_files = 'Providers/Intercom/AnalyticsKitIntercomProvider.swift'
+    i.frameworks = 'Intercom'
+    i.dependency 'AnalyticsKit/Core'
+    i.pod_target_xcconfig = {
+      'FRAMEWORK_SEARCH_PATHS' => '$(inherited) $(PODS_ROOT)/Intercom/Intercom',
+      'OTHER_LDFLAGS'          => '$(inherited) -undefined dynamic_lookup'
+    }
+  end
+
+  s.subspec 'Crashlytics' do |c|
+    c.source_files = 'Providers/Crashlytics/AnalyticsKitCrashlyticsProvider.swift'
+    c.frameworks = 'Crashlytics'
+    c.dependency 'AnalyticsKit/Core'
+    c.pod_target_xcconfig = {
+      'FRAMEWORK_SEARCH_PATHS' => '$(inherited) $(PODS_ROOT)/Crashlytics/iOS',
+      'OTHER_LDFLAGS'          => '$(inherited) -undefined dynamic_lookup'
```

Comment:
This fixes the issue where someone that uses this library also includes Crashlytics as a dependency and CocoaPods complains about the transitive static dependency conflict.